src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -147,11 +147,15 @@ C_FUNC(\Name\()_End):
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
         .ifnc \pushArgRegs, DoNotPushArgRegs
-                PUSH_ARGUMENT_REGISTERS
+                push {r0 - r3}
+                .save {r0 - r3}
         .endif
 
-        PUSH_CALLEE_SAVED_REGISTERS
-        
+        push {r4 - r11, lr}
+        .save {r4 - r11, lr}
+
+        PROLOG_STACK_SAVE r7
```

Comment:
This is the required action to allow unwind through this frame if this frame calls another function. .save are recommended for unwinder to further understand the register saved.